src/utils/dockerflow.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable header/header */
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import fs from "fs";
+import path from "path";
+import AppConstants from "../appConstants.js";
+import packageJson from "../../package.json";
+
+
+const versionJsonPath = path.join(__dirname, "../../version.json");
+
+
+// If the version.json file already exists (e.g., created by circle + docker),
+// don't need to generate it
+if (!fs.existsSync(versionJsonPath)) {
+  const versionJson = {
+    source: packageJson.homepage,
+    version: packageJson.version,
```

Comment:
Probably not this exact line, but looks like the `version` isn't getting set properly in stage/prod by circle + docker:
```js
{
  source: "https://github.com/mozilla/blurts-server",
  version: "1.0.0",
  NODE_ENV: "production"
}
```

Reply:
We will have to look into this